modules/custom/az_bef_extend/js/az-bef-extend.js

```diff
@@ -0,0 +1,16 @@
+(function (Drupal) {
```

Comment:
Javascript for this should probably be written as ES6 and transpiled to ES5 via our [javascript transpilation process](https://github.com/az-digital/az_quickstart/blob/main/CONTRIBUTING.md#compiling-javascript-in-local-development)